gpii/node_modules/gpii-oauth2/gpii-oauth2-datastore/test/js/DataSource-pouchDB.js

```diff
@@ -0,0 +1,154 @@
+/*!
+Copyright 2016 OCAD university
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+/* eslint-env browser */
+/* eslint strict: ["error", "function"] */
+
+var fluid = fluid || require("infusion");
+var gpii = fluid.registerNamespace("gpii");
+
+(function () {
+
+    "use strict";
+
+    // The data source is to support the in-browser testing using pouchDB as the backend data storage.
+    // It overrides default setImpl() and getImpl() provided by "kettle.dataSource.URL" with gpii.pouch API calls.
+    fluid.defaults("gpii.dataSource.pouchDB", {
```

Comment:
This very useful dataSource should have its own tests. Also, I don't see a reason for it to be in the "test" directory or indeed within gpii-oauth at all. This should be moved to a top-level subproject within gpii/node_modules so that it can later be incorporated into Kettle or gpii-pouchdb as appropriate. 

Doesn't this DataSource do a substantial part of the work of express-pouchdb (only at the client rather than the server end)?